docs/adrs/0004-field-for-institution-information.md

```diff
@@ -0,0 +1,68 @@
+
+---
+
+### Other options that were discussed are described below:
+
+**Option 1 - Remap Aardvark field `schema_provider_s` to `TimdexRecord.source`**
+
+As of this writing, the `source` field derives its values from the names defined in the variable `transformer.config.SOURCE`. These names are hard-coded values that define the source of the TIMDEX resource at a higher level. This approach proposes storing lower-level values for OGM resources specifically, deviating from how the field is interpreted by other TIMDEX sources. 
+
+Considerations for this approach:
+* `MITAardvark` transformer would need to override `JSONTransformer.get_required_fields`
+* `TimdexRecord.source` is a required field, but it would map data from `schema_provider_s`, which--although always derived by `geo-harvester`--_technically_ isn't required by the Aardvark metadata schema.
+   * Not an issue at a technical level (i.e., can be done), but confusing at a conceptual level.
+
+
+**Option 2 - Remap Aardvark field `schema_provider_s` to `TimdexRecord.holdings`**
+
+This approach proposes mapping the values from `schema_provider_s` to `TimdexRecord.holdings` under one of two floated fields to the `Holding` data model: `location` or `institution`.
+
+Considerations for this approach:
+* As of this writing, `holdings` are only derived by the `Marc` transformer (for Alma resources).
+* `Holding.location`: Existing values for this field include the physical location for a resource (e.g., "Library Storage Annex").
+   * This approach would result in the values for `location` having different levels of specificity (e.g., host institution is not the same as physical location), which will be confusing.
+* `Holding.institution`: This would be a new field, comprised of a single string value, that is added to the `Holding` data model.
+   * There may be some resources that are aggregated by multiple institutions. The concept of "data provider" vs. "provider" was discussed. 
+        * **Provider**: defined as "the institution / organization that hosts or provides access to the resource"
+        * **Data provider**: the institution / organization that provided the metadata record you see before you
+
+## Consequences
+
+For GIS records, the UI can directly reference the values from `TimdexRecord.institution` to display the host institution for a resource on [Geodata](https://geodata.libraries.mit.edu/).
+
+For other TIMDEX sources, given the decision outlined above, there should be little to no effect. It could be beneficial to revisit these sources and see if there is a field we can map to that was previously ignored.
```

Comment:
I'm slightly nervous that we don't know yet of other sources will use this field. I'm okay if this is only a useful field for GIS at this time, but the nervousness comes from not knowing if we want to move other things here that indeed make the label of it less good (i.e. `Institution` is perfect for GIS, but `metadata_provider` would work for both GIS and additional sources if we considered mappings more broadly before adding this field. Or not!).

I'd really prefer if we could do the investigation to be able to take a stronger stance here. Either "we don't think this field will be used by any of our current sources so there is no risk" or "we think we'd map fields X from MARC and Y from DSpace and they work perfectly with this plan".

Reply:
Thanks for articulating this @JPrevost, I agree.

Reading this ADR, it felt like a top level, single string field was flexibile enough that we could both modify how it's stored in the data model, and how it's retrieved in the API layer, as needed.  But if it's immediate goal is to support the GIS data, as a thought experiment, why don't we call it `gis_data_resource_provider`?  Obviously, that's too specific; so what then _do_ we mean by `institution`?  are we expecting other sources to potentially use this?

Though I already approved, I think the ADR would benefit from discussion about what `institution` means, and how other sources may, or may not, use it.  Going to leave some inline comments, now that I reread it through this lens.